source/deployment/enterprise-deployment-guide.rst

```diff
@@ -205,10 +209,10 @@ Notifications
 
 Notifications have gained importance in our daily lives. Modern operating systems all have a way to point the user's attention towards important events from specific apps. There are three different types of notifications in Mattermost: desktop, email, and mobile push notifications. Mattermost will notify you of messages with any of these characteristics:
 
-- Direct Messages
```

Comment:
This list should have full stops/periods at the end of each line; none of the lines is a full sentence.

Reply:
See first comment on this. Agree that they're not full sentences, will revert re: style.